tdms/src/iterator.cpp

```diff
@@ -2455,8 +2455,8 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[]) {
 
     if ((sourcemode == sm_steadystate) && (runmode == rm_complete) && exphasorsvolume) {
 
-      extractPhasorsVolume(E, E_s, dft_counter, *omega_an, *dt, Nsteps);
-      extractPhasorsVolumeH(H, H_s, dft_counter, *omega_an, *dt, Nsteps);
+      E.set_phasors(E_s, dft_counter - 1, *omega_an, *dt, Nsteps);

```

Comment:
Note for reviewers: Maybe the electric field phase shouldn't have the `n + 1` thus needing this `-1` at the start of the loop.